Automattic/wp-calypso
# Show Awesome Motive (WPBeginner) plugins for partner bundle sites

Related to https://github.com/Automattic/wp-calypso/pull/95551

## Proposed Changes

* This replaces the Paid plugins section with Awesome Motive (WPBeginner) plugins if a site was created using the parter bundle signup.

## Why are these changes being made?


* This is needed for peRNfl-14g-p2

## Testing Instructions


* Log in
* Signup using this flow `/setup/new-hosted-site?partnerBundle=wpbeginner-special`
* You should see the WPBeginner featured plugins on top.

<img width="1275" alt="Screenshot 2024-10-25 at 16 28 47" src="https://github.com/user-attachments/assets/b03281b8-053e-40d5-86a4-67d869431137">



## Pre-merge Checklist
